examples/workspace/example1.md

```diff
@@ -1,5 +0,0 @@
-= Documentation =
```

Comment:
We shouldn't delete this file as it is needed for testing the navigation to external files from a diagram.

Reply:
readded